source/docs/software/wpilib-tools/robot-characterization/analyzing-data.rst

```diff
@@ -0,0 +1,43 @@
+Analzing Data
+=============
+
+Once we have data from a characterization run, we can analyze it. To launch the data analyzer, click on the ``Launch Data Analyzer`` button.
+
+.. image:: images/launching-data-analyzer.png
+   :alt: Launching the data analyzer
+
+Loading your Data File
+----------------------
+
+Now it's time to load the data file we saved from the logger tool. Click on ``Select Data File``.
+
+.. image:: images/selecting-data-file.png
+   :alt: Selecting the existing data file
+
+In the resulting file dialog, select the JSON file you want to analyze. If the file appears to be malformed, an error will be shown.
+
+Running Feedforward Analysis
+----------------------------
+
+Once a data file has been selected, the ``Analyze Data`` button becomes available in the ``Feedforwad Analysis`` frame. Click it.
```

Comment:
```suggestion
Once a data file has been selected, the ``Analyze Data`` button becomes available in the ``Feedforward Analysis`` frame. Click it.
```
`FeedForward` or `Feedforward`?

Reply:
In the program itself, it's labelled as ``Feedforward``